app/models/project.rb

```diff
@@ -0,0 +1,100 @@
+# == Schema Information
+#
+# Table name: projects
+#
+#  id              :integer          not null, primary key
+#  name            :string(255)
+#  model_id        :integer
+#  origin_id       :integer
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  target_size     :integer
+#  current_size    :integer
+#  comments        :text
+#  admin_user_id   :integer
+#  project_type_id :integer
+#
+
+class Project < ActiveRecord::Base
+    include Elasticsearch::Model
+
+  belongs_to :country, foreign_key: 'origin_id'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.